plugins/microsoft_teams/help.md

```diff
@@ -120,405 +121,410 @@ Example output:
-    "@odata.context": "https://graph.microsoft.com/beta/$metadata#teams('xxxx')/channels('xxxxxxx')/messages/$entity",
-    "id": "1571769574238",
-    "etag": "1571769574238",
-    "messageType": "message",
-    "createdDateTime": "2019-10-22T18:39:34.238Z",
-    "importance": "normal",
-    "locale": "en-us",
-    "webUrl": "https://teams.microsoft.com/l/message/xxxxxxxxxxxxxx/xxxxxx?groupId=xxxxx",
-    "from": {
-      "user": {
-        "id": "xxxxxxx",
-        "displayName": "Test User",
-        "userIdentityType": "aadUser"
-      }
-    },
-    "body": {
-      "contentType": "text",
-      "content": "Hello from InsightConnect!"
-    },
-    "attachments": [],
-    "mentions": [],
-    "reactions": []
-  }
+  "success": true
 }
 ```
 
-#### Send Message by GUID
-
-This action sends a message using the GUID for the team and channel. This is more performant than send message.
+#### Create Teams Enabled Group
+  
+Create a group in Azure and enable it for Microsoft Teams
 
 ##### Input
 
 |Name|Type|Default|Required|Description|Enum|Example|
-|----|----|-------|--------|-----------|----|-------|
-|channel_guid|string|None|True|Channel GUID|None|xxxxx-xxxxx-xxxx-xxxx|
-|is_html|boolean|None|True|Is the message HTML|None|True|
-|message|string|None|True|Message to send|None|Hello!|
-|team_guid|string|None|True|Team GUID|None|xxxxx-xxxxx-xxxx-xxxx|
-
+| :--- | :--- | :--- | :--- | :--- | :--- | :--- |
+|group_description|string|None|True|Group description|None|A test group|
+|group_name|string|None|True|Team name|None|test_group|
+|mail_enabled|boolean|None|True|Should e-mail should be enabled for this group|None|True|
+|mail_nickname|string|None|True|The nickname for the email address of this group in Outlook|None|TestGroup|
+|members|[]string|None|False|A list of usernames to set as members|None|["user@example.com"]|
+|owners|[]string|None|False|A list of usernames to set as owners|None|["user@example.com"]|
+  
 Example input:
 
 ```
 {
-  "channel_guid": "xxxxx-xxxxx-xxxx-xxxx",
-  "is_html": false,
-  "message": "Hello!",
-  "team_guid": "xxxxx-xxxxx-xxxx-xxxx"
+  "group_description": "A test group",
+  "group_name": "test_group",
+  "mail_enabled": true,
+  "mail_nickname": "TestGroup",
+  "members": "user@example.com",
+  "owners": "user@example.com"
 }
 ```
 
 ##### Output
 
-|Name|Type|Required|Description|
-|----|----|--------|-----------|
-|message|message|False|The message object that was created|
-
-Example input:
+|Name|Type|Required|Description|Example|
+| :--- | :--- | :--- | :--- | :--- |
+|group|group|False|Information about the group that was created|None|
+  
+Example output:
 
 ```
 {
-  "channel_guid": "xxxxx-xxxxx-xxxx-xxxx",
-  "is_html": false,
-  "message": "Hello!",
-  "team_guid": "xxxxx-xxxxx-xxxx-xxxx"
+  "group": {
+    "Created Date Time": {},
+    "Description": {},
+    "Display Name": {},
+    "ID": {},
+    "Mail": {},
+    "Mail Enabled": {},
+    "Mail Nickname": "",
+    "Security Enabled": "true"
```

Comment:
Could you please include some examples? 😄

Reply:
I think large objects are unreadable in help.md form and it's optional from a validator perspective